.github/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 ### Improvements
 
+* Changing deprecated abstract property definitions
```

Comment:
Relatively minor, but we try and keep the changelog in the past tense (so that it makes more sense when reading past entries)
```suggestion
* Removed deprecated `@abstractproperty` decorators
```